examples/embedded/src/main/java/org/eclipse/jetty/embedded/FastFileServer.java

```diff
@@ -183,6 +183,12 @@ public void failed(Throwable x)
                     x.printStackTrace();
                     async.complete();
                 }
+
+                @Override
+                public InvocationType getInvocationType()
+                {
+                    return InvocationType.NON_BLOCKING;
+                }
```

Comment:
There are at least 20 times that we override and implement this method exactly as this.   Perhaps it would be better to have a `Callback.NonBlocking` interface that had this method as default?

Reply:
Hmmm I tried this in my PR and I don't think it added to code clarity.  So forget this.